lib/hammer_cli_foreman/host.rb

```diff
@@ -173,14 +173,10 @@ def get_power_status
 
     class PuppetRunCommand < HammerCLIForeman::SingleResourceCommand
       command_name "puppetrun"
-      resource :puppet_hosts
-      action :puppetrun
```

Comment:
Actually, whilst this is still present in API docs, there is no need to remove that. The same with `build_options`.

Oltherwise the command looks empty (which is kind of OK since we deprecate it, just looks weird).